### ISSUE TITLE
No accent or non-native-speaker axis in the TTS matrix

### ISSUE THREAD

**Issue by trsdn** [trsdn/asr-bench#12]
Every voice in the pool is a standard-accent native speaker of its language. Accent and non-native speech are among the largest real-world WER drivers — commonly a bigger effect than the noise profiles the harness already models — and the benchmark currently has no axis for them at all.

This is a coverage gap rather than a bug, but it limits what any result here can claim. A model that looks robust across `clean`→`worst-case` has been shown robust to *channel* degradation only.

Options, roughly in order of effort:

- Piper's voice catalogue includes regional variants for several languages; a `de_DE` script rendered with an `en_US`-trained voice speaking German is a crude L2 proxy, but a real one.
- Some multi-speaker checkpoints expose accent conditioning directly.
- Recording real non-native speakers would be the honest version and gives up the word-exact ground truth that makes this harness cheap — probably a separate, smaller validation set rather than part of the matrix.

Worth deciding explicitly whether this belongs in scope. If it does not, the README should say so, since "synthetic sessions with a known transcript" currently reads as more general than it is.

**Comment by trsdn** [comment]
Partially addressed, with an explicit caveat about what the proxy does and does not measure.

`conversations/accent-en.yaml` is the `allhands-en` conversation with identical words, timing and overlap, except three of the six speakers are voiced by **German Piper models reading English text**. A German VITS voice phonetises English with German grapheme-to-phoneme rules, which puts the substitutions roughly where an L2 German speaker puts them.

The design point is the **within-session** contrast: both groups share one recording, one diarisation and one ASR model, and the reference is byte-identical, so the voice is the only variable.

| Speakers | cpWER |
|---|---:|
| a, c, e — native English voices | **11.2 %** |
| b, d, f — German voices reading English | **78.1 %** |

Seven times worse from the voice alone. Whole session 12.4 % → 49.6 % cpWER; undiarised baseline 5.6 % → 38.9 % WER.

The failure is legibly phonetic:

```
REF  I can walk through it quickly. At two nineteen the primary queue
     stopped draining and nothing alerted for eleven minutes.
HYP  E canvalk truk it quickly. At twoninitin, be primachi quay stoppet
     draining and noting alatet for eleven mining eleven.
```

`through` → `truk`, `the` → `be`, `nothing alerted` → `noting alatet`, `walk` → `valk` — the *th*, *w* and vowel substitutions German speakers actually make. Then the stream truncates: the model does not merely mis-hear the accent, it gives up.

**Diarisation was almost untouched** — DER 11.4 % against 8.0 % on the native session, and all six speakers still got their own cluster. Accent damages *what was said*, not *who said it*. The two halves of the pipeline fail on different things, which is another argument for selecting them independently.

### What this does not measure

78 % is not a light accent, and the issue title asks for an accent axis rather than a worst case. A German TTS model has never been trained on English, so it applies German phonology unconditionally, where a real L2 speaker applies it partially and inconsistently. It also reproduces none of the prosody, hesitation or syntactic transfer that a second-language speaker brings, and it is far more uniform than any human accent.

So the number is an **upper bound on accent difficulty**. It answers two questions decisively — *does an accent axis move the results?* and *does it hit diarisation or transcription?* — and it does not answer *what does a mild accent cost*.

Leaving this open for the honest version, which needs recorded L2 speech rather than synthesis. Options, roughly in order of effort: a permissively licensed L2 corpus, deliberate per-voice phoneme perturbation, or accepting that the synthetic route tops out here and treating the accent axis as binary (native / heavy) rather than graded.
